docs/useIntersection.md

```diff
@@ -0,0 +1,36 @@
+# `useIntersection`
+
+React sensor hook that tracks the changes in the intersection of a target element with an ancestor element or with a top-level document's viewport.
```

Comment:
Can be useful to add "...using Intersection Observer API" with a link to https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API as the hook returns a complete intersection observer entry object?